switch_model/transmission/transport/build.py

```diff
@@ -217,9 +224,6 @@ def define_components(mod):
         mod.TRANSMISSION_LINES,
         within=NonNegativeReals,
         default=1)
-    mod.trans_capital_cost_per_mw_km = Param(
-        within=NonNegativeReals,
-        default=1000)
     mod.trans_lifetime_yrs = Param(
         within=NonNegativeReals,
         default=20)
```

Comment:
Agree with bumping this to a higher value by default for future runs.

Reply:
Replying to this and your previous comment:

- This is the default from the upstream SWITCH
- It should never impact our results since we always specify the value
- I prefer not to change this since this is the default from the upstream switch and it doesn't affect us. All I did was move the definition further up in the code so it's properly defined.